dhis2/src/main/resources/webapp/js/controllers.js

```diff
@@ -35,6 +37,14 @@
             };
 
             $scope.submit = function () {
+                if(!$scope.settings.serverURI.match("^(ht|f)tps?:\/\/.+")) {
+                    if($scope.selectedProtocol.id == 3) {
```

Comment:
If we give them a dropdown for the protocol, we can assume that they won't enter it in the url. Eventually we can strip the protocol part from the input.  

On a side note, we don't have dropdowns for the protocol part anywhere in Motech. If we go with this, we should create a ticket for making the url input consistent.